src/collections/_documentation/platforms/javascript/advance-settings.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Advanced Settings
+sidebar_order: 30001
+---
+
+## BrowserClient
+
+To be able to manage several Sentry instances without any conflicts between them you need to create your own `Client`.
+This also helps to prevent tracking of any parent application errors in case if your application is integrated
+inside of it.
+
+```javascript
+import { BrowserClient } from "@sentry/browser";
+
+const client = new Sentry.BrowserClient({
```

Comment:
```suggestion
const client = new BrowserClient({
```

Reply:
Just found out that @HazAT already fixed this example